packages/nimble-components/src/select/template.ts

```diff
@@ -81,21 +87,49 @@ SelectOptions
             horizontal-positioning-mode="locktodefault"
             horizontal-scaling="anchor"
             ?hidden="${x => (x.collapsible ? !x.open : false)}">
-            <div
-                class="listbox"
-                id="${x => x.listboxId}"
-                part="listbox"
-                role="listbox"
-                ?disabled="${x => x.disabled}"
-                ${ref('listbox')}
-            >
-                <slot
-                    ${slotted({
+            <div class="listbox-background">
+                <div
+                    class="
+                        listbox 
+                        ${x => (x.filteredOptions.length === 0 ? 'empty' : '')}
+                        ${x => x.positionAttribute}
+                    "
+                    id="${x => x.listboxId}"
+                    part="listbox"
+                    role="listbox"
+                    ?disabled="${x => x.disabled}"
+                    ${ref('listbox')}
+                >
+                    ${when(x => x.filterMode !== FilterMode.none, html<Select>`
+                        <div class="search-field ${x => x.positionAttribute}">
```

Comment:
I think it would be good to standardize on the term "filter", so this would be `class="filter-field"`, and the icon would be `class="filter-icon"`. Thoughts?